### PR TITLE
Replace use of "upgrade" with "update"

### LINKED ISSUE #31397: The term "upgrade" is being used incorrectly and interchangeably with "update"
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

When viewing the update all page, some buttons reference update while others incorrectly reference upgrade. Normally, upgrade involves a transaction, or a higher tier (normally paid) of a certain product. Update means the version has been improved and an update needs applied to the base product. I noticed this when I went to the Update All page, and then hesitated when I saw the "Upgrade All" button, thinking, it can't be possible for me to upgrade all these modules to a higher (often paid) tier service, the person must have meant Update All. When I clicked the Upgrade All button, it did indeed "Update" all the modules.

Screenshot 2023-02-16 045320

I thought at first that this was a typo in the button itself, so I went exploring the code documentation and discovered that Upgrade is used incorrectly in multiple other locations, including the containing folder for the code itself! This is far more than a typo, so I'm here writing a bug instead.

image
image

This is a larger refactoring project that I'm not skilled enough to undertake.

### Expected behavior

Update means updating the base version.
Upgrade means applying something more than the "base" version provides (often paid).

### Steps to reproduce

1. Module Manager
2. Updates
3. Look for "Update All"
4. Look for documentation on how to Update
5. Find interchanging references to Update and Upgrade
6. End up trying the button Upgrade All
7. The Upgrade All prompts notifications that the modules have been _Updated_

### PrestaShop version(s) where the bug happened

8.0.1

### PHP version(s) where the bug happened

7.4.33

### If your bug is related to a module, specify its name and its version

Module Manager

### PR DESCRIPTION
**Pull request by eternoendless**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Replaces the use of "upgrade" with "update"
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | The wordings are updated as per [this request](https://github.com/PrestaShop/PrestaShop/issues/31397#issuecomment-1434781963). 
| Fixed ticket?     | Fixes #31397
| Related PRs       | N/A
| Sponsor company   | PrestaShop SA

### CODE REVIEW

**Review by kpodemski — DISMISSED** [review]

**Review by boherm — DISMISSED** [review]

**Review by lartist — DISMISSED** [review]

File: src/PrestaShopBundle/Resources/views/Admin/Module/updates.html.twig [hunk 1]
```diff
@@ -35,7 +35,7 @@
         {% include '@Common/HelpBox/helpbox.html.twig' with { 'content' : "Update these modules to enjoy their latest versions."|trans({}, 'Admin.Modules.Help'), 'title' : "Modules to update"|trans({}, 'Admin.Modules.Feature') } %}
 
         {% if (modules|length > 1) and (level >= constant('PrestaShop\\PrestaShop\\Core\\Security\\Permission::LEVEL_UPDATE')) %}
-          <a class="btn btn-primary-reverse float-right btn-outline-primary light-button module_action_menu_upgrade_all" href="#" data-confirm_modal="module-modal-confirm-upgrade-all">{{ "Upgrade All"|trans({}, 'Admin.Modules.Feature') }}</a>
+          <a class="btn btn-primary-reverse float-right btn-outline-primary light-button module_action_menu_upgrade_all" href="#" data-confirm_modal="module-modal-confirm-upgrade-all">{{ "Update All"|trans({}, 'Admin.Modules.Feature') }}</a>
```
Comment (l-delin) [inline src/PrestaShopBundle/Resources/views/Admin/Module/updates.html.twig]:
```suggestion
          <a class="btn btn-primary-reverse float-right btn-outline-primary light-button module_action_menu_upgrade_all" href="#" data-confirm_modal="module-modal-confirm-upgrade-all">{{ "Update all"|trans({}, 'Admin.Modules.Feature') }}</a>
```

**Review by l-delin — COMMENTED** [review]
Hello and thank you!
There's just a quick edit to make.

Also, I noticed the string `'You are not allowed to upgrade this module.'` (Admin.Modules.Notification) seems to be missing. 
It should be `'You are not allowed to update this module.' `Could you please add it? Thanks!

**Comment by eternoendless** [comment]
> Also, I noticed the string `'You are not allowed to upgrade this module.'` (Admin.Modules.Notification) seems to be missing. 
> It should be `'You are not allowed to update this module.' `Could you please add it? Thanks!

I would gladly do it, but it seems this wording isn't used anywhere in the code.

**Review by boherm — APPROVED** [review]

**Comment by l-delin** [comment]
got it @eternoendless, thank you! 

Wording ✔️

**Review by kpodemski — APPROVED** [review]

**Review by Hlavtox — APPROVED** [review]

**Review by MhiriFaten — COMMENTED** [review]
Hello @eternoendless ,

I tried to test your PR and installed previous versions of modules so that they are found in the updates part to check the **`Upgrade All`** button.
But no module with old version is found in the updates part.

Here is a screen record : 
 
[Untitled_ Apr 26, 2023 11_36 AM.webm](https://user-images.githubusercontent.com/111756615/234536732-433fb853-21d3-4f6c-acf8-973836efbe2c.webm)

--
We have here : 

> **Mail alerts v2.3.2** ( ps_mailalerts : latest version **v2.4.1**)
> **Best customers v2.0.3** (statsbestcustomers : latest version **v2.0.4**)

Could you check it please ?

**Comment by eternoendless** [comment]
Hey @MhiriFaten I think that's because the distribution API is not providing anything for 8.1 yet. 

I think you can try to hack it away by changing the version on your PrestaShop install to a previous one like 8.0.0 so you get results from the API.

You can do that by editing [this file](https://github.com/PrestaShop/PrestaShop/blob/develop/src/Core/Version.php#L35-L38) and setting `VERSION` to "8.0.0" and `MINOR_VERSION` to "0"

**Review by MhiriFaten — APPROVED** [review]
Hello @eternoendless ,

Thank you for your explanation, I modified the version of the develop to 8.0.0 with minor_version: 0, and I found the modules to update.

The 'upgrade' is replaced by 'update'

image

Automated tests are OK.

It is QA approved :heavy_check_mark: 

Thank you

**Comment by kpodemski** [comment]
thanks everyone

**Pull request merged**